react/features/filmstrip/components/Filmstrip.web.js

```diff
@@ -89,11 +91,8 @@ class Filmstrip extends Component {
  * }}
  */
 function _mapStateToProps(state) {
-    const { remoteVideosVisible } = state['features/filmstrip'];
-    const { disable1On1Mode } = state['features/base/config'];
-
     return {
-        _remoteVideosVisible: Boolean(remoteVideosVisible || disable1On1Mode)
+        _remoteVideosVisible: shouldShowRemoteVideos(state)
```

Comment:
I feel uncomfortable about this. It looks like by _remoteVideosVisible we're interested simply if the remote videos in the filmstrip are currently shown or hidden. So it looks like, the filmstrip itself should do some calculations internally based on some actions and expose a state like the one we have for _filmstripVisible, instead of exposing a function.

Reply:
I felt uncomfortable about that suggested change though. Essentially all pieces for knowing this state is in the redux store and it can be derived.

Reply:
Confirmed with @lyubomir that this is indeed a good practice in our project and there's no problem with it! So apparently it was just me feeling uncomfortable with this without a good reason.